tests/scripts/test_i_dependabot_auto_merge.py

```
@@ -0,0 +1,139 @@
+    prs = i.fetch_open_dependabot_prs()
+    assert len(prs) == 1
+    assert prs[0].number == 42
+    assert prs[0].head_ref_name == "dependabot/pip/requests-2.1"
+    assert prs[0].mergeable_state == "clean"
+
+
+def test_fetch_open_dependabot_prs_exits_on_gh_failure(monkeypatch):
+    monkeypatch.setattr(i, "run_gh", lambda args: _FakeResult(1, "", "boom"))
+    try:
+        i.fetch_open_dependabot_prs()
+        assert False, "expected SystemExit"
```

Ruff (0.15.21)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)